crates/rolldown/tests/rolldown/topics/new_url/opt-out/main.js

```diff
@@ -0,0 +1 @@
+/* @rolldown-ignore */ new URL('path-should-not-exists', import.meta.url);
```

Comment:
```suggestion
new URL(/* @rolldown-ignore */ 'path-should-not-exists', import.meta.url);
```

What's the correct pattern @sapphi-red ?